app/services/user_opportunity_discovery.py

```
@@ -2281,10 +2486,306 @@ def _calculate_options_risk(self, greeks: Dict[str, float]) -> str:
+                        },
+                        discovered_at=self._current_timestamp(),
+                    )
+                )
+
+            self.logger.info(
+                "✅ Volatility trading scanner completed",
+                scan_id=scan_id,
+                opportunities=len(opportunities),
+            )
+
+        except Exception as exc:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)